docs/Conformance-Guide-draft.md

```diff
@@ -0,0 +1,103 @@
+
+   The extender needs to document where the user can find this information in the conformance application.
+
+1. (Best Practice) User settings consistency: **For a consistent user experience we recommend that user settings and configuration settings follow the naming conventions as documented in the Zowe Explorer extensibility documentation.**
+
+   The list and documentation of the configuration settings should be made available as part of the compliance application. If the extension is a VS Code extensions then including the package.json file will be sufficient unless other means of configuration settings (such as environment variables) have been used.
+
+1. (Best Practice) Error message consistency: **For a consistent user experience error and log messages should follow the format documented in the Zowe Explorer extensibility documentation.**
+
+   To verify this requirement it is sufficient to provide links to user documentation showing screen shots of error dialogs as well as sample log file entries.
+
+1. (Best Practice) Zowe SDK usage: **Utilize available Zowe SDKs to standardize on z/OS interactions as well as other common capabilities used by many other Zowe extensions and tools.**
+
+   This can be verified by reviewing the dependencies list of the implementation such as in the package.json file of a nodejs-based extension.
+
+## Profiles Access extension conformance criteria (1)
+
+A Profiles Access extension is a Zowe Explorer extension that uses the Zowe Extensibility API to conveniently access Zowe CLI profiles loaded by Zowe Explorer itself. This allows the extension to consistently access profile instances of specific types, offer them for edit and updates as well as common refresh operations that apply to all extensions, add more profile types it is using itself for its own custom views (for example a CICS extension adding a CICS explorer view) and other similar use cases related to Zowe CLI profiles. These extensions do **not** have to be VS Code extension if it just wants to use ProfilesCache implementation of Zowe Explorer as all APIs are provided free of any VS Code dependencies. Such an extension could be used for another non-VS Code tool, a Zowe CLI plugin, a Web Server or another technology. However, to access the profiles cache of the actual running VS Code Zowe Explorer the extender needs to be a VS Code extension that has an extension dependency defined to be able to query the extender APIs. Therefore, some of the criteria that are listed here as required are only required if the extender is a VS Code extension.
+
+1. (Required if VS Code) VS Code extension dependency: **Zowe Explorer VS Code extensions can only be activated after Zowe Explorer is fully activated itself. Therefore, to ensure the correct activation order and extender must include an extensionDependencies to Zowe Explorer entry in their package.json file.**
```

Comment:
Extension defines Zowe Explorer as a VSCode extension dependency (Code Example). It includes an extensionDependencies to Zowe Explorer entry in their package.json file and it is only activated once Zowe Explorer is fully activated.

Reply:
I believe defining Zowe Explorer in the extender's `extensionDependencies` would ensure that the extender is only activated once Zowe Explorer is activated. So perhaps we could shorten the bolded lines to:

"Extension defines Zowe Explorer as a VSCode extension dependency (Code Example). It includes an extensionDependencies entry for Zowe Explorer in its package.json file."

Then, we can move the explanation of the rationale into the paragraph below: "This requirement ensures the correct activation order, with the extender only activating once Zowe Explorer is fully activated."